sys/netgraph/ng_pipe.c

```diff
@@ -771,7 +771,7 @@ pipe_dequeue(struct hookinfo *hinfo, struct timeval *now) {
 		 * the original packet...
 		 */
 		if (hinfo->cfg.duplicate &&
-		    random() % 100 <= hinfo->cfg.duplicate) {
+		    arc4random() % 100 <= hinfo->cfg.duplicate) {
```

Comment:
Why arc4random() instead of prng32() or prng32_bounded()? The former is a cryptographic PRNG, which doesn't look to be what you want.

Reply:
Changed as well as the other `random(9)` replacement.